### PR TITLE
WINDUPRULE-172

### PR DESCRIPTION
**Pull request by OndraZizka**
Check individual commits, there is some formatting, if that causes trouble I can separate.

Open for discussion, I was following common sense but I may be missing past Marc's wishes/slides/mails/comments.

See also https://github.com/windup/windup/pull/1130

### CODE REVIEW

File: rules-reviewed/eap6/weblogic/weblogic.windup.xml [hunk 1]
```diff
@@ -22,11 +22,14 @@
                 </javaclass>
             </when>
             <perform>
-                <classification issue-display-mode="all" title="WebLogic scheduled job" effort="3" category-id="mandatory">
```
Comment (mareknovotny) [inline rules-reviewed/eap6/weblogic/weblogic.windup.xml]: category-id is removed intentionally?

File: rules-reviewed/eap6/weblogic/weblogic.windup.xml [hunk 2]
```diff
@@ -35,6 +38,8 @@
                       6 Tutorial](https://docs.oracle.com/javaee/6/tutorial/doc/bnboy.html).
                       ]]>
                     </message>
+                    <link href="https://docs.oracle.com/javaee/6/tutorial/doc/bnboy.html" title="EJB Timer Service" />
```
Comment (mareknovotny) [inline rules-reviewed/eap6/weblogic/weblogic.windup.xml]: there is already a link in hint message, the same URL but different title ? it looks weird in the report
screenshot from 2017-06-16 12-52-54

Comment (jsight) [inline rules-reviewed/eap6/weblogic/weblogic.windup.xml]: I agree... I will remove this link while merging.

**Comment by jsight** [comment]
This seems good... will merge it now.

**Pull request merged**